pkg/apis/apps/validation/validation_test.go

```diff
@@ -127,6 +127,37 @@ func TestValidateStatefulSet(t *testing.T) {
 					}()},
 			},
 		},
+		{
+			ObjectMeta: metav1.ObjectMeta{Name: "abc-123", Namespace: metav1.NamespaceDefault},
+			Spec: apps.StatefulSetSpec{
+				PodManagementPolicy: apps.ParallelPodManagement,
+				Selector:            &metav1.LabelSelector{MatchLabels: validLabels},
+				Template:            validPodTemplate.Template,
+				Replicas:            3,
+				UpdateStrategy: apps.StatefulSetUpdateStrategy{
+					Type: apps.RollingUpdateStatefulSetStrategyType,
+					RollingUpdate: func() *apps.RollingUpdateStatefulSetStrategy {
+						return &apps.RollingUpdateStatefulSetStrategy{Partition: 2, MaxUnavailable: func() *intstr.IntOrString {
+							numberTwo := intstr.FromInt(2)
+							return &numberTwo
+						}()}
+					}()},
+			},
+		},
+		{
```

Comment:
Isn't this test case repeated above?

Reply:
@alculquicondor  this one doesnt have maxUnavailable, so that will assume a value of 1

Reply:
yes, but isn't this tested in lines 116-128?

Reply:
The PodManagementPolicy on line 119 is `PodManagementPolicy: apps.OrderedReadyPodManagement,`